src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -20,18 +20,11 @@ public abstract class AuthenticationManager
 
         public abstract IEnumerable<AuthenticationDescription> GetAuthenticationSchemes();
 
-        public abstract Task AuthenticateAsync(AuthenticateContext context);
+        public abstract Task<AuthenticateInfo> GetAuthenticateInfoAsync(string authenticationScheme);
```

Comment:
That's why I think a simpler name like `GetUserAsync` or `GetPrincipalAsync` would be better for the "get just the principal" use case (people often mix up `AuthenticateAsync` and `SignInAsync`).

Anyway, as long as the odd `AuthenticateAsync` overload is removed, I'm fine :smile: